app/src/main/res/values/static_strings.xml

```diff
@@ -15,4 +15,7 @@
     <string name="about_debug_menu_toast_progress" translatable="false">Debug menu: %1$d click(s) left to enable</string>
     <string name="about_debug_menu_toast_done" translatable="false">Debug menu enabled</string>
 
+    <!-- Preference for developers -->
```

Comment:
nit: From just this comment it's unclear what the string represents and where it it used.

Reply:
Done